plugins/epos/plugin.py

```diff
@@ -100,9 +98,7 @@ def get_metadata(self):
         # leave this here for a while until we make sure everthing works
         metadata_sample = []
         eml_schema = "epos"
-        final_url = (
-            "https://www.ics-c.epos-eu.org/api/v1/resources/details?id=" + self.item_id
-        )
+        final_url = self.oai_base + self.item_id
```

Comment:
`oai_base` shall be the root path of the EPOS API i.e. "https://www.ics-c.epos-eu.org/api/v1". The path "/resources/details?id=" should be added by the EPOS plugin